desc/efit.py

```diff
@@ -0,0 +1,374 @@
+"""Class for reading and processing EFIT files."""
+
+import numpy as np
+from matplotlib import pyplot as plt
+from scipy.interpolate import CubicSpline
+
+from desc.basis import DoubleFourierSeries
+from desc.equilibrium import Equilibrium
+from desc.geometry import FourierRZToroidalSurface
+from desc.grid import LinearGrid
+from desc.profiles import PowerSeriesProfile, SplineProfile
+from desc.transform import Transform
+
+
+class EFITIO:
+    """Performs input from EFIT files to DESC Equilibrium."""
+
+    @classmethod
+    def load(  # noqa: C901 - fxn too complex
+        cls,
+        g,
+        M,
+        N=0,
+        L=None,
+        bdry_dist=1,
+    ):
+        """Create a DESC equilibrium object from an EFIT geqdsk file.
```

Comment:
probably a stupid quesiton, but in the inputs you have a `eqdsk_cocos.out` file, is that the same as a `geqdsk` file?

Reply:
Yes, it's an eqdsk file. The data may have been preprocesses but the format is exactly the same as an eqdsk file.